src/Middleware/NodeServices/src/HostingModels/NodeInvocationException.cs

```diff
@@ -56,4 +56,4 @@ public NodeInvocationException(string message, string details, bool nodeInstance
             AllowConnectionDraining = allowConnectionDraining;
         }
     }
-}
+}
```

Comment:
Seems like this file should be removed from this PR, as there is no meaningful change here.

Reply:
Not sure how that got in. I've reverted the changes made to this file.